features/tests/snapshots/snap_test_api.py

```diff
@@ -131,3 +131,39 @@
         }
     }
 }
+
+snapshots["test_feature_opening_hours 1"] = {
+    "data": {
+        "features": {
+            "edges": [
+                {
+                    "node": {
+                        "properties": {
+                            "openingHours": [
+                                {
+                                    "openingHours": [
```

Comment:
Just a matter of opinion I suppose, but personally I find it confusing if there are fields with the same name containing different things.

Reply:
That is true, it was so in the specification. I'll probably have a discussion about this still.